### PR TITLE
Frontendmango

### PR DESCRIPTION
**Pull request by oneacik**
Yoooooooo

### CODE REVIEW

File: front/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+node_modules
+dist/bundle.js
+dist/index.html
+.idea
```
Comment (allgreed) [inline front/.gitignore:4]: Plz use global `.gitignore` for files generated by your tooling of choice

File: front/src/backend/server.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+(async function main(): Promise<void> {
+    const app: Express = express();
+    const port = 3000;
+
+    app.use(express.json());
+    app.use(cors());
+
+    const compiler = webpack(webpackConf);
+
+    app.use(webpackMiddleware(compiler));
```
Comment (oneacik) [inline front/src/backend/server.ts:17]: That's a good shit.

File: front/src/frontend/view/Main.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                public openness_to_experience: number
+    ) {
+    }
+}
+
+
+export class Main
+    extends React.Component<{}, {}> {
+
+    bigFive = ["extroversion", "neuroticism", "agreeableness", "conscientiousness", "openness_to_experience"];
```
Comment (oneacik) [inline front/src/frontend/view/Main.tsx:31]: Try to remove dependency by extracting it from class fields. (Is it even possible?)

File: front/src/frontend/view/Main.tsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+                name: '', email: '', sex: 'Female',
+                ...Object.fromEntries(this.bigFive.map((x: any) => [x, 50]))
+            }}
+            onSubmit={(values: any, actions: any) => {
+                this.submit(values);
+
+            }}
+        >
+            {(props: any) => (
+                <form onSubmit={props.handleSubmit}>
```
Comment (oneacik) [inline front/src/frontend/view/Main.tsx:81]: create helper functions or components for encapsulating those fields.

File: front/src/test/e2e/selenium.test.ts [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    await findAndWrite(driver, 'email', 'oneat@vp.pl');
+    driver.findElement(By.name('Male')).click();
+
+    await findAndWrite(driver, 'extroversion', '21');
+    await findAndWrite(driver, 'neuroticism', '37');
+    await findAndWrite(driver, 'agreeableness', '42');
+    await findAndWrite(driver, 'conscientiousness', '69');
+    await findAndWrite(driver, 'openness_to_experience', '96');
+
+    await driver.findElement(By.name('submit')).click();
```
Comment (oneacik) [inline front/src/test/e2e/selenium.test.ts:56]: Add waiting condition after.

**Pull request merged**